new/postinstall.sh

```diff
@@ -27,6 +29,15 @@ do
 	mysql -u patchdemo -ppatchdemo patchdemo_$NAME < $PATCHDEMO/sql-perwiki/$sql
 done
 
+# OOUI build
+if [ -d $PATCHDEMO/wikis/$NAME/w/build/ooui ]; then
+	cd $PATCHDEMO/wikis/$NAME/w/build/ooui
+	npm install
+	npm x grunt build
```

Comment:
Annoyingly this also builds the PHP demos (including a composer install) but this will need to be fixed upstream.

Reply:
Turns out we can use the demos, so this isn't a waste of time: #432